plugin/trino-redshift/pom.xml

```diff
@@ -173,6 +173,30 @@
     </dependencies>
 
     <profiles>
+        <profile>
+            <id>default</id>
```

Comment:
The other option would be to add in `ci.yml`

```
!:trino-redshift,
```

after 

https://github.com/trinodb/trino/blob/11942a4f918b8d079ddcdfa3fec3d7f53b3ab9da/.github/workflows/ci.yml#L447

I'm inclined however to use rather the current option because it gives us extra test-coverage for non cloud-tests that are executed this way on `trino-redshift`.